runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -1383,7 +1383,9 @@ MM_CopyForwardScheme::mainSetupForCopyForward(MM_EnvironmentVLHGC *env)
 	_phantomReferenceRegionsToProcess = 0;
 
 	/* Sort all hot fields for all classes as dynamicBreadthFirstScanOrdering is enabled */
-	MM_HotFieldUtil::sortAllHotFieldData(_javaVM, _extensions->globalVLHGCStats.gcCount);
+	if (_extensions->scavengerScanOrdering == MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST) {
```

Comment:
Would you please put constant first?
```suggestion
	if (MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST == _extensions->scavengerScanOrdering) {
```

Reply:
Yes my mistake, was too hasty making this change.